packages/checkbox/src/css/index.ts

```diff
@@ -114,11 +114,6 @@ export default {
     fill: colorsTextIcon.highOnDark
   },
 
-  // __input
-  '.psds-checkbox__input': {
-    display: 'none'
```

Comment:
this completely removed the input from the accessibility tree

Reply:
why did we have it, then?

Reply:
we want to hide the input - but i used the screen ready only styles instead